apps/api/src/app/account/create-account.dto.ts

```diff
@@ -17,6 +17,14 @@ export class CreateAccountDto {
   @IsString()
   currency: string;
 
+  @IsOptional()
+  @IsString()
+  id: string;
+
+  @IsOptional()
```

Comment:
I would remove `isDefault`, because you could easily create multiple default accounts.